docs/rst/container_yml/reference.rst

```diff
@@ -427,8 +427,8 @@ which means:
 secrets
 .......
 
-Supported by the ``deploy`` command. The secrets directive be used to define a literal, file or from a directory to be mounted into
-a container. If supplied secrets are not base64 encoded then they are encoded when the deployment playbook is generated.
+Supported by the ``deploy`` and ``run`` commands. The secrets directive can be used to associate a secret with a container, mounting it as
```

Comment:
If we are not supporting Docker at first, the `run` command would not support secrets correct?